docs/reference/migration/migrate_3_0.asciidoc

```diff
@@ -285,6 +285,11 @@ The two individual setters for existence() and nullValue() were removed in favou
 optional constructor settings in order to better capture and validate their interdependent
 settings at construction time.
 
+==== NotQueryBuilder
+
+The NotQueryBuilder which was deprecated in 2.1.0 is removed. As a replacement use BooleanQueryBuilder
```

Comment:
BoolQueryBuilder ? ;) can you include an example of what we used to have and what you have after the migration?